### DISCUSSION TITLE
What models work well with Plandex and Ollama?

### DISCUSSION THREAD

**Discussion by mhamzahkhan** [plandex-ai/plandex#204] [General]
I'm playing with Ollama and Plandex. I tried a few different models but I struggled with the models I tried.

I was wondering which models you guys have had success with whilst using a self-hosted LLM

**Comment by danenania** [comment]
Hey @mhamzahkhan, check this out: https://docs.plandex.ai/models/model-providers#limitations

I'm hoping to put some more work into this area with the latest oss models to see where they stand, but I think they are likely still lacking in some key areas that Plandex relies on.
